kie-assets-library-assets/pom.xml

```diff
@@ -121,6 +121,23 @@
                                     </copy-resources>
                                 </config>
                             </project-definition>
+                            <project-definition>
+                                <id>pmml-generated</id>
```

Comment:
I suspect the naming here might be affected by the example definitions (dmn-generated). It is no hard rule and in fact it would be much more useful if the naming here would reflect the purpose/origin. `pmml-kie-benchmarks` would be a perfectly fine naming IMO. Or naming based on the domain that the models deal with.

Reply:
@jstastny-cz 
Thank you for the review. I've changed the names to be more appropriate.